controllers/core/event_controller.go

```diff
@@ -174,7 +176,7 @@ func (r *EventReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl
 					}
 					if labelled {
```

Comment:
Later in this PR we handle the !updated case. Won't we have a similar situation here?

Reply:
Since the label node method returns `if updated label` and `any error`, here we have checked err and returned result if err != nil. When the `if updated label` == false but no error, we know the node has the label key/value pair and just update the prometheus metrics.
```
eventReconcileOperationLatency.WithLabelValues(ReconcileEventsLatencyMetrics).Observe(float64(time.Since(start).Milliseconds()))
```

In another place, the `!updated` is used for logging purpose
```
else if !updated {
					b.log.V(1).Info("unable to update node with the existing label key/value pair", "NodeName", instance.Name(), "LabelKey", config.HasTrunkAttachedLabel, "ExpectedLabelValue", labelValue)
```